canonical/cloud-init
# cloud.cfg.tmpl: make sure "centos" settings are identical to "rhel"

## Proposed Commit Message
```
We have a couple of bugs where centos does not have the default user as rhel.
This PR makes sure the configuration is exactly the same.

Signed-off-by: Emanuele Giuseppe Esposito <eesposit@redhat.com>

```
RHBZ: 2115565 and 2115576


## Test Steps
1. Boot the latest CentOS Stream cloud image
2. Attempt to log in with the "cloud-user"
3. Attempt to log in with the "centos" user

Actual results:
Login fails with "cloud-user" and succeeds with "centos"

Expected results:
Login succeeds with "cloud-user" and fails with "centos"

Additional info:
Originally, RHEL/CentOS used "cloud-user" with the custom cloud-init configuration files. Too much configuration was removed in the rebase to cloud-init v22.1.

## Checklist:
